File: Iteration 3/CourseSection.py
from abc import ABC, abstractmethod
from Course import Course
from User import User

class CourseSection:
    def __init__(self, day, place, hour):
        self.day = day
        self.place = place
        self.hour = hour

    def get_day(self):
        return self.day

    def get_place(self):
        return self.place

    def get_hour(self):
        return self.hour

    def __str__(self):
        return f"CourseSection{{place='{self.place}', day='{self.day}', hour='{self.hour}'}}"

File: Iteration2/CSE3063F24P1_GRP14/src/CourseSection.java
import com.fasterxml.jackson.annotation.JsonProperty;

import java.util.List;

class CourseSection {

    @JsonProperty("day")
    private String day;

    @JsonProperty("place")
    private String place;

    @JsonProperty("hour")
    private String hour;

    public CourseSection() {

    }

    // Constructor
    public CourseSection(String day, String place, String hour) {
        this.day = day;
        this.place = place;
        this.hour = hour;

    }


    // Getters
    public String getDay() {
        return day;
    }

    public String getPlace() {
        return place;
    }

    public String getHour() {
        return hour;
    }


    // toString method for printing
    @Override
    public String toString() {
        return "CourseSection{" +
                "place='" + place + '\'' +
                ", day='" + day + '\'' +
                ", hour=" + hour +
                '}';
    }
}